gpflow/params/dataholders.py

```diff
@@ -60,13 +57,7 @@ def trainable(self):
     def parameter_tensor(self):
         return self._dataholder_tensor
 
-    @property
-    def shape(self):
```

Comment:
Why is this property removed? I see that `batch_size` is added, but what if we want to find the input dimension?

Reply:
it is inherited from `parameter` class